### ISSUE TITLE
support for "work profile"

### ISSUE THREAD

**Issue by vbatts** [OpenLauncherTeam/openlauncher#516]
#### General Information

* **OpenLauncher Version:** 0.7.0 ( https://f-droid.org/app/com.benny.openlauncher)
* **Android Version:** 9
* **Custom ROM:** No
* **Install Source:** F-Droid

I see https://github.com/OpenLauncherTeam/openlauncher/issues/282 was closed a while ago but the release pushed to fdroid 2wks ago does not seem to include support for work profile. How do I access this feature?

#### Log

This is only needed for crashes or other issues you notice in your system logs from OpenLauncher.

**Comment by dkanada** [comment]
I tried out an implementation but it broke a lot of things so it was reverted.

**Comment by vbatts** [comment]
I'll beta test for you

-------- Original Message --------

**Comment by ivysrono** [comment]
+1

**Comment by frknkrc44** [comment]
My test device: LG Q6, using Android 8.1

I tried and works now, but I have some problems:
- If I don't enabled work profile, work profile intents don't loading at first time.
- Work profile apps don't have any indicator for show that's in work profile or work profile is disabled.
- I only see one icon for any app even I installed second one of same app in work profile.

**Comment by vbatts** [comment]
Same.

(Thanks for bringing in the initial support!)

-------- Original Message --------

**Comment by JonnyTech** [comment]
I am on the latest version from fdroid and see no option to show work profile apps. As missing something? There is nothing in settings.

**Comment by dkanada** [comment]
It's working on the master branch but install and uninstall events don't trigger a refresh, so I haven't pushed an update to F-Droid yet.

**Comment by Bernie** [comment]
@dkanada Is that to say that this is not yet implemented in the released version 0.7.2?  Is there any consideration in releasing an update that has this new support, even if it does not yet update the application list automatically?  Even with this limitation, the added support for work profile apps would be very helpful.

**Comment by dkanada** [comment]
Yes, I was planning on going through with a release soon, so you'll see it in the next one.
